app/assets/stylesheets/global.scss

```diff
@@ -66,3 +66,34 @@ span.small {
 #map_canvas {
   height: 30rem;
 }
+
+.tranRecord {
+  border-right: groove black;
+  margin-right: 5px;
+}
+
+.bankForm {
+  margin: auto;
+  width: 50%;
+}
+
+.drakeBank {
+  width: 4em;
+  border-radius: 40px;
+}
+
+.characterDropDown {
+  width: 75%;
+  margin: auto;
+}
+.bank {
+  margin-left: 5px;
+}
+
+.characterSelect {
+  width: 50%;
+}
+
+.startingBalance {
+  width: 50%;
+}
```

Comment:
I am suspicious of using CSS for this, especially relative widths. `MyCrossroads` is using the Foundation for Sites CSS framework; you could accomplish a similar effect by dropping your code inside of some grid elements, e.g.:
```html
<div class="row">
  <div class="small-6 columns end">
    <!–– YOUR CODE GOES HERE ->
  <div>
</div>
```

Reply:
Updated to use much more Foundation, with a little CSS on the side.